scripts/gen_FCHL19_rep.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/python3
+import argparse
+import os
+import sys
+
+import numpy as np
+from ase.io import read, write
```

Comment:
@BingqingCheng Are we treating ase and asaplib as third-party libraries or project-level?

Reply:
What is the difference between the two?

Reply:
There'll be a difference in the import block in which they're included. Third-party libraries are in a separate block to project-level imports.